### PR TITLE
First Draft - Create and Rename Item

### PR DESCRIPTION
**Pull request by vhuaytav**
Added methods and unit tests for create/rename item
Also correct wrong behavior with delete single file

### CODE REVIEW

File: src/Utilities/BasicOperation.java [hunk 1]
```diff
@@ -140,14 +140,45 @@ public static boolean moveItem(File source, File target) {
...
+    }
+    
+    /**
+     * Method to rename an item, it works for single file or directory
+     * @param oldItem File that contains the path and old name of the file.
+     * @param newItem File that contains the path and the new name of the file.
+     * @return true if the file with the new name exists
+     */
+    public static boolean renameItem(File oldItem, File newItem) {
+        oldItem.renameTo(newItem);
```
Comment (edson-gonzales) [inline src/Utilities/BasicOperation.java:181]: what happens with the old file, is this deleted?

File: src/Utilities/BasicOperation.java [hunk 2]
```diff
@@ -140,14 +140,45 @@ public static boolean moveItem(File source, File target) {
...
+    
+    /**
+     * Method to create a single directory in a path specified.
+     * @param name String that contains the name of the directory to be created.
+     * @param path File with the path where the new directory will be created.
+     * @return true if the directory was created without errors, false if it the directory already exists.
+     */
+    public static boolean createDirectory(String name, File path){
+        File newDirectory = new File (path.getAbsolutePath(), name);
+        boolean result = false;
```
Comment (edson-gonzales) [inline src/Utilities/BasicOperation.java:167]: If the directory exists it will return false is this correct?

**Comment by edson-gonzales** [comment]
looks good please review the observations that i made, correct them and you can merge it with the main branch

**Comment by vhuaytav** [comment]
for the second observation: yes, it returns false when the directory already exists in the system.

**Pull request closed without merging**